products/sle/main.pm

```diff
@@ -918,6 +918,20 @@ elsif (get_var("VIRT_AUTOTEST")) {
         loadtest "virt_autotest/guest_migration_dst";
     }
 }
+elsif (get_var("PERF_KERNEL")) {
+    if (get_var("PERF_INSTALL")) {
+        load_boot_tests();
+        load_inst_tests();
+    }
+    elsif (get_var("PERF_SETUP")) {
+        loadtest "kernel_performance/login_console";
+        loadtest "kernel_performance/install_qatestset";
+    }
+    elsif (get_var("PERF_RUNCASE")) {
+        loadtest "kernel_performance/login_console";
+        loadtest "kernel_performance/run_perf_case";
+    }
```

Comment:
@asdil12 do you have a recommendation how this could be e.g. extracted into an extratest?